src/couch_replicator_connection.erl

```diff
@@ -120,9 +124,8 @@ handle_cast({connection_close_interval, V}, State) ->
 % owner crashed
 handle_info({'DOWN', _Ref, process, Pid, _Reason}, State) ->
```

Comment:
But not if owner exits with reason `normal` and somehow never relinquishes the workers